CHANGELOG.md

```diff
@@ -1,5 +1,8 @@
 # CHANGELOG
 
+## Unreleased
+* Support NetworkAttachmentType and LaunchMode default updates to the ImageCapabilitySchema [GH-46]
+ 
```

Comment:
CHANGELOG entries are now autogenerated on release and included as the release notes for the plugin [release](https://github.com/hashicorp/packer-plugin-oracle/releases). I've updated the title of this PR to match the CHANGELOG entry. Once release it will get added to the release notes. 
```suggestion
```